### ISSUE TITLE
Refactor pm2 monit + pm2 logs

### ISSUE THREAD

**Issue by Nicholaiii** [Unitech/pm2#2631]
If reporting a bug, please use the following template.

#### Expected behaviour
Expecting app to redraw and conform to sizes.

#### Actual behaviour
![Image of bug](http://puu.sh/tlXew/9f0d6e48d1.png)

#### Steps to reproduce

- `pm2 m`
- resize terminal

#### Software versions used

```
OS         : Ubuntu 16.04
node.js    : 6.9.1
PM2        : 2.2.3
```

#### PM2 Log output

Use the command: `tail --lines 50 ~/.pm2/pm2.log`

```
Not applicable; but if you need it I will paste it.
```

**Comment by vmarchaud** [comment]
We actually use this module to do this but its really old, we may rework `pm2 monit` to achieve this.

**Comment by Nicholaiii** [comment]
I made a quick, working solution using [blessed](https://github.com/chjj/blessed/#blessed), a really handy and flexible terminal interface API. It handles redrawing like a charm.
[gist here](https://gist.github.com/Nicholaiii/51592c356d577ffc5c6b27ae0f3f4aed)
Feel free to steal as much or little as you want to. :)

**Comment by vmarchaud** [comment]
The aim will be to make a much complete UI, but thanks for your gist anyway :)

**Comment by Unitech** [comment]
Could be great to re work the pm2 logs command also
=> Sample work from @soyuka https://github.com/keymetrics/pm2-logs

**Comment by soyuka** [comment]
\o/ old work! IMO blessed is not supported enough to be a stable tool that we could use... This is why we dropped that pm2 solution it was too buggy in some terminals no?

**Comment by Unitech** [comment]
Great work from @ItsJimi landed in #development branch!

Try out the new dashboard/monitoring for PM2:

```bash
$ npm install Unitech/pm2#development -g
$ pm2 monit
```

Will land in PM2@2.4

**Comment by Unitech** [comment]
Look at this!:

![screenshot from 2017-02-02 17-19-57](https://cloud.githubusercontent.com/assets/757747/22557830/00fe5ffe-e96c-11e6-85f4-5bfe962bfbbd.png)

**Comment by Unitech** [comment]
@ItsJimi

- [ ] Add some padding inside boxes (between box and text)
- [ ] Add interpreter in Metadata box
- [ ] Move the Code metrics panel just below the process list and metadata below the global log panel - the process list and code metrics panel must have the same width

**Comment by vmarchaud** [comment]
Published PM2 `2.4.0`
